doc/augmentations/badphotocopy.md

```diff
@@ -29,3 +14,59 @@ augmentation = BadPhotocopy(
 | `p`             | The probability this augmentation will be applied.               |
 
 Click here for an [example Colab Notebook](https://colab.research.google.com/drive/1_n7atcI-xVSpnV17YuzZhd9ISe735fBo?usp=sharing)
+
+**Example Usage:**
```

Comment:
@proofconstruction I changed the structure of doc a bit to move the example code down since the example section will be quite long now.

Reply:
Good idea